src/Raven.Server/Documents/Handlers/Debugging/ServerWideDebugInfoPackageHandler.cs

```diff
@@ -209,8 +198,6 @@ public async Task GetClusterWideInfoPackage()
         [RavenAction("/admin/debug/info-package", "GET", AuthorizationStatus.Operator, IsDebugInformationEndpoint = true)]
         public async Task GetInfoPackage()
         {
-            var stacktraces = StackTraces();
```

Comment:
Why we removed this?

Reply:
We use the endpoint instead of local function now